File: C#JAVA_Client_Server/ClientServcerC#Protocols/Solution1/Domain/Validators/UtilizatorValidator.cs
using System;
using Domain.validators.interfaceAndException;

namespace ConsoleApp1.domain.validators
{
    public class UtilizatorValidator : IValidator<Utilizator>
    {
        public void validate(Utilizator entity)
        {
            if (entity.NumeUtilizator.Length == 0 )
            {
                throw new Exception ("Utilizator INVALID!");
            }
        }
    }
}

File: C#JAVA_Client_Server/ClientServcerC#Protocols/LongExample/Domain/src/main/java/entities/domain/validator/UtilizatorValidator.java
package entities.domain.validator;


import entities.domain.Utilizator;
import entities.domain.validator.interfaceAndExceptions.ValidationException;
import entities.domain.validator.interfaceAndExceptions.Validator;

public class UtilizatorValidator implements Validator<Utilizator> {
    @Override
    public void validate(Utilizator entity) throws ValidationException {
        if(entity.getNumeUtilizator().length()==0)
            throw new ValidationException("entities.domain.Utilizator invalid!");
    }
}
